partner_contact_configuration/__manifest__.py

```diff
@@ -7,15 +7,15 @@
     'summary': """
         Adds menu configuration access through the 'contacts' module main menu
         """,
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
     'license': 'AGPL-3',
     'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
     'website': 'https://acsone.eu',
-    'depends': ['base',
-                'contacts'
-                ],
-    'data': ['views/partner_contact.xml'
-             ],
-    'demo': [
+    'depends': [
+        'base',
```

Comment:
No need for base

Reply:
I'd prefer explicit dependencies

Reply:
I do not agree in this case, but not blocking.